src/lib/slide-toggle/slide-toggle.ts

```diff
@@ -160,16 +160,17 @@ export class MatSlideToggle extends _MatSlideToggleMixinBase implements OnDestro
 
   /**
    * An event will be dispatched each time the slide-toggle input is toggled.
-   * This event always fire when user toggle the slide toggle, but does not mean the slide toggle's
-   * value is changed. The event does not fire when user drag to change the slide toggle value.
+   * This event is always emitted when the user toggles the slide toggle, but this does not mean
+   * the slide toggle's value has changed. The event does not fire when the user drags to change
+   * the slide toggle value.
    */
   @Output() readonly toggleChange: EventEmitter<void> = new EventEmitter<void>();
 
   /**
    * An event will be dispatched each time the slide-toggle is dragged.
-   * This event always fire when user drag the slide toggle to make a change that greater than 50%.
-   * It does not mean the slide toggle's value is changed. The event does not fire when user toggle
-   * the slide toggle to change the slide toggle's value.
+   * This event is always emitted when the user drags the slide toggle to make a change greater
+   * than 50%. It does not mean the slide toggle's value is changed. The event is not emitted when
+   * the user toggles the slide toggle to change the slide toggle's value.
```

Comment:
"slide toggle to change the slide toggle's value" -> "slide toggle to change its value"

Reply:
Okay, I made that change.